amethyst_assets/src/prefab/system.rs

```diff
@@ -102,26 +106,32 @@ where
                     let new_entity = entities.create();
                     self.entities.push(new_entity);
                     if let Some(parent) = entity_data.parent {
-                        parents
-                            .insert(
-                                new_entity,
-                                Parent {
-                                    entity: self.entities[parent],
-                                },
-                            ).unwrap();
+                        if let Err(e) = parents.insert(
+                            new_entity,
+                            Parent {
+                                entity: self.entities[parent],
+                            },
+                        ) {
+                            error!("Unable to insert new entity with a `Parent`: {}", e);
+                        }
+                    }
+                    if let Err(e) = tags.insert(
+                        new_entity,
+                        PrefabTag::new(prefab.tag.expect("Prefab tag is not present")),
```

Comment:
Would this happen and not be a bug? If it is always a bug, could you mention it in the expect?

Reply:
Hmm I looked at it a lot before I decided to just log because from what it seemed like in the in definitions of `Prefab`, it seems to be a lot of `None`s all over the place before there is inserted an actual value. Hence I was actually going to ask for some help on this one.

Reply:
You should get the input of @Rhuagh, he wrote the prefab system.

Reply:
This should not be able to fail, since every loaded prefab has such a tag. If the prefab is not loaded, it cannot be retrieved at this point.